### ISSUE TITLE
what event can i call to get the current view window min/max

### ISSUE THREAD

**Issue by lrossy** [angular-google-chart/angular-google-chart#281]
Im trying to solve the following situation:

Chart is drawn, with options.explorer: { actions: ['dragToZoom', 'rightClickToReset'] } and the ability to hide a series by clicking on the legend. 

Whenever the options change in the chart, the chart is redrawn. So, if a user were to zoom in via the explorer, then click on a series to hide or show, the chart is redrawn without any knowledge of the previous zoom setting, causing it to zoom back out to the initial zoom. 

Im thinking if i can fetch the new hAxis.min and max when the click on the legend, then i can apply those settings in the hideSeries() method so it redraws with the same range.

**Comment by nbering** [comment]
I'll have to look closer to familiarize myself with that part of the API to give advice. I always just ignored explorer as a use case while doing the refactor to the current version. Makes it too complicated to reason about the wrapper.

You can access the chartWrapper from any of the even directive callbacks though, so you may be able to get the options, maybe not.

**Comment by lrossy** [comment]
You're right i can access the options through the chartWrapper, however it seems that when you select a range using explorer it does not seem to save that data to the options. So im not sure exactly how to maintain the explorer values when the chart is redrawn. Thanks for the idea though, ill keep digging at this if you can think of anything else let me know.

**Comment by nbering** [comment]
If I remember right there may be a way to get hold of an explorer object. You might give that a try.
